bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,241 @@
+          <li><img src="{{ static('img/firefox/os-new/icons/browser.png') }}" width="142" height="142" class="connected entertained fade" alt="{{_('Browser')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/clock.png') }}" width="142" height="142" class="organized fade" alt="{{_('Clock')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/contacts.png') }}" width="142" height="142" class="connected organized fade" alt="{{_('Contacts')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/music.png') }}" width="142" height="142" class="entertained fade" alt="{{_('Music')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/fmradio.png') }}" width="142" height="142" class="entertained fade" alt="{{_('FM Radio')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/settings.png') }}" width="142" height="142" class="organized fade" alt="{{_('Settings')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/gallery.png') }}" width="142" height="142" class="entertained organized fade" alt="{{_('Gallery')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/usage.png') }}" width="142" height="142" class="organized fade" alt="{{_('Usage')}}" /></li>
+          <li><img src="{{ static('img/firefox/os-new/icons/marketplace.png') }}" width="142" height="142" class="connected entertained organized fade" alt="{{_('Marketplace')}}" /></li>
+        </ul>
+      </div>
+    </section>
+    <section class="fxos-marketplace">
+      <div class="content-wrapper">
+        <h2>{{_('Add more from the Firefox Marketplace')}}</h2>
+        <p>{{_('Get all the most popular games and social networking apps right here.')}}</p>
+        <ul class="apps-marketplace">
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/facebook.png') }}" width="142" height="142" alt="Facebook" />
+            <span>Facebook</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/youtube.png') }}" width="142" height="142" alt="YouTube" />
+            <span>YouTube</span>
+          </li>
+          <li>
+            <img src="{{ static('img/firefox/os-new/icons/line.png') }}" width="142" height="142" alt="Line" />
+            <span>Line</span>
+          </li>
+        </ul>
+        <a href="//marketplace.firefox.com/" class="cta-button marketplace">{{_('Go to the Marketplace')}}</a>
+        <p class="footnote">{{_('* Pre-installed apps may vary by manufacturers and operators.')}}</p>
+      </div>
+    </section>
+
+    {% set feed_links = firefox_os_feed_links(LANG) %}
+    {% if feed_links %}
+    <section class="fxos-news">
+      <div class="content-wrapper">
+        <aside>
+          <h2>{{_('In the news')}}</h2>
+          <p><a href="">{{_('See more')}}</a></p>
```

Comment:
This is on my 'things I still need to do' list ;) https://bugzilla.mozilla.org/show_bug.cgi?id=1120700#c50